CHANGELOG

```diff
@@ -1,3 +1,25 @@
+Changes in cclib-1.6.4
+
+Features:
+    * [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
+    * [GSOC2020] New methods: bickelhaupt, stockholder, hirshfeld and ddec calculate partial charges (Minsik Cho)
+    * [GSOC2020] New bridge to Horton (Minsik Cho)
+    * [GSOC2020] Support reading cube files in volume method (Minsik Cho)
+    * New bridge to Atomic Simulation Environment (Felipe S. S. Schneider)
+    * New bridge to PySCF (Amanda Dumi)
+    * New attribute dispersionenergies for molecular dispersion energy corrections (Theo Keane)
```

Comment:
This is actually originally from Jonathon for ORCA, and the remaining implementations are from me.

Reply:
Ah, it was more that I was expecting no name to be present (following the above convention). Will we have issue filers for major features/bug fixes in the list now? If so, can you add it to https://github.com/cclib/cclib/issues/555?

Reply:
No, you're right... reverted these. Thanks for keeping an eye on these things. Please LMK if I misrepresented something else.